.github/workflows/CI.yml

```diff
@@ -44,20 +44,16 @@ jobs:
   docs:
     name: Documentation
     runs-on: ubuntu-latest
+    timeout-minutes: 40
     steps:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1.6' # for now we will be using 1.6 for Documentation
+          version: '1.6'
+
       - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-docdeploy@v1
+      - run: julia --project=docs -e 'include("docs/make.jl")'
```

Comment:
The docdeploy has something fancy for GitHub logging. Unfortunately, that didn't work well together with Turing's logging and/or Pluto's logging.

Reply:
Turing and Pluto do a lot of macro and finicky things...